### PR TITLE
ch3: fix typo in Object.setPrototypeOf() perf note

### PR DESCRIPTION
**Pull request by vsemozhetbyt**
`` `obj.__proto__ = …` `` -> `` `Object.setPrototypeOf(...)` ``.

Sync it with the first sidebar paragraph and the [source](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/setPrototypeOf).

### CODE REVIEW

**Comment by vsemozhetbyt** [comment]
Closing as it seems this repo does not accept PRs anymore.

**Comment by bevacqua** [comment]
It does, just took me a bit to get back to them

File: ch03.asciidoc [hunk 1]
```diff
@@ -877,7 +877,7 @@ Using `Object.setPrototypeOf` to change the prototype of an object is an expensi
...
+Changing the prototype of an object is, by the nature of how modern JavaScript engines optimize property accesses, a very slow operation, in every browser and JavaScript engine. The effects on performance of altering inheritance are subtle and far-flung, and are not limited to simply the time spent in a `Object.setPrototypeOf(...)` statement, but may extend to any code that has access to any object whose prototype has been altered. If you care about performance you should avoid setting the prototype of an object. Instead, create a new ((("objects", "built-in improvements", startref="ob3bii")))object with the desired ((("objects", "setting prototypes", startref="ob3sp")))((("Object.setPrototypeOf", startref="ospo3")))prototype using `Object.create()`.
```
Comment (bevacqua) [inline ch03.asciidoc]: Can we use `…` instead of `...`?

Comment (vsemozhetbyt) [reply ch03.asciidoc]: Done)

**Pull request merged**